content/docs/tasks/security/authn-policy.md

```diff
@@ -249,6 +249,62 @@ $ kubectl exec $(kubectl get pod -l app=sleep -n legacy -o jsonpath={.items..met
 200
 ```
 
+## Enable mTLS incrementally
```

Comment:
Please try to use "mutual TLS" so others can easily understand it.

Reply:
And same below

Reply:
Done.